### DISCUSSION TITLE
how to modify the workflow.

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#16909] [How to]
want to autofill the contact in the quotes.  

#create workflow and choose the quote module
#choose update the fields
# choose contact=  [expression]  select contactid from vtiger_contactdetails where parentid=accountid and charge like '%Quotation%'

but this does not work....anyone can help?

**Comment by mariuszkrzaczkowski** [comment]
this syntax is not supported in workflow
